src/services/doppler-legacy-client.ts

```diff
@@ -452,4 +471,14 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
       };
     }
   }
+
+  public async getBannerData(lang: string, type: string, page: string) {
+    const response = await this.axios.get(
+      `https://qa.fromdoppler.com/wp-json/doppler2019/v1/getbanner?filter[lang]=${lang}&filter[type]=${type}&filter[page]=${page}`,
+    );
+    if (!response || !response.data) {
+      throw new Error('Empty Site response');
```

Comment:
What do you think about making this method fail-safe and catch all errors and always return a valid _Promotions_ value?